Tasks/AzureFileCopy/Utility.ps1

```diff
@@ -1410,11 +1373,45 @@ function Add-AzureVMCustomScriptExtension
 function Check-ContainerNameAndArgs
 {
     param([string]$containerName,
-          [string]$additionalArguments)
+          [string]$additionalArguments,
+          [bool]$useDefaultArgumentsForBlob)
```

Comment:
this parameter is not used anywhere. If not required, then remove

Reply:
done